source/Calamari.Common/Plumbing/FileSystem/FileOperations.cs

```diff
@@ -110,6 +116,31 @@ public IEnumerable<string> EnumerateDirectoriesRecursively(string path)
             return Directory.EnumerateDirectories(path, "*", SearchOption.AllDirectories);
         }
 
+        public virtual IEnumerable<string> EnumerateFiles(
```

Comment:
My apologies if I've missed them, but this and the other new methods in this class are prime candidates for unit tests.

Reply:
(I like these, btw 😁)

Reply:
As per our Slack conversation, I'm ok with you adding unit tests for these in a separate PR, given that this is only currently used by commands being a feature toggle.

Reply:
Any chance we can do those tests in a subsequent PR? I think this should be ok as it's just passing the buck to the built-in methods anyways but yeah happy to add some tests afterwards.